.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 isort mypy pyupgrade safety
+      - run: bandit --recursive --skip B101,B108,B110,B310,B311,B403,B404,B603 .
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="ans,kno,lof,te,thisy" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E501,E9,F63,F7,F82 --max-complexity=18
+                      --max-line-length=215 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt || true
+      # - run: pip install -e .
+      - run: mkdir -p .mypy_cache
+      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
```

Comment:
same, for all the remaining tests, limit the scope to `python/`

Reply:
All of these tests should be completed in a minute or two max.  All these tools have better algorithms for finding files that contain Python code and can process the entire repo in seconds.  We these tools to find bugs in our examples and utilities before our users do.

Reply:
It's mainly about noise. In practice, `build/` and `3rdparty_downloads/` will be created during C++ compilation. LInters will exam 3rd-party Python code unless we strict the directory. 

You're right that the examples shall be covered. Then, could you restrict the directory to `examples/python` and `python`?